src/components/LandingPage/Promo/GitStarPromoSection/useGitStars.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react'
+import { getStarCount } from './getStarCount'
+import { getStarFromLocalStorage } from './getStarFromLocalStorage'
+
+export const useGitStars = () => {
+	const [starCount, setStarCount] = useState(0)
+
+	useEffect(() => {
+		try {
+			const localStorageStarCount = getStarFromLocalStorage()
+			if (localStorageStarCount !== null) {
+				setStarCount(localStorageStarCount.starCount)
+			}
+			if (localStorageStarCount === null) {
```

Comment:
Not needed to check the null value when the previous if statement exists - this is just redundant.

Reply:
It was added due to TS error

Reply:
Just add the return statement after the setStarCount function invocation.